ledger-core/virtual/handlers/vdelegatedcallrequest.go

```diff
@@ -114,9 +114,9 @@ func (s *SMVDelegatedCallRequest) stepProcessRequest(ctx smachine.ExecutionConte
 	)
 
 	action := func(state *object.SharedState) {
-		if state.GetState() != object.HasState {
-			panic(throw.IllegalState())
-		}
+		// if state.GetState() != object.HasState {
```

Comment:
remove this >_<

Reply:
sure, but that commit would be partially overriden by https://github.com/insolar/assured-ledger/pull/331